backend/p8mobility.dbup/scripts/Bus.sql

```diff
@@ -0,0 +1,12 @@
+START TRANSACTION;
+
+CREATE TABLE IF NOT EXISTS Bus
+(
+    Id        VARCHAR(40) PRIMARY KEY,
+    Latitude  DECIMAL(10, 8),
+    Longitude DECIMAL(11, 8),
+    Action    ENUM ('Default', 'Accelerate', 'Decelerate','MaintainSpeed') DEFAULT 'Default',
```

Comment:
Burde der ikke kun være tre states, for hvad skal default være? At bussen holder stille eller?

Reply:
Den er til når man logger ind og en bus skal have en initial state